### PR TITLE
feat: add `dtm` spec

### PR DESCRIPTION
**Pull request by imxw**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
`dtm` command are not currently supported

**What is the new behavior (if this is a feature change)?**
Autocompletion support for `dtm`

**Additional info:**
`dtm` project found [here](https://github.com/devstream-io/devstream)

### CODE REVIEW

File: src/dtm.ts [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+        },
+      ],
+    },
+
+    {
+      name: "help",
+      description: "Help about any command",
+      args: {
+        name: "command",
+        generators: cmds,
```
Comment (fedeci) [inline src/dtm.ts:225]:
```suggestion
        templates: "help",
```

File: src/dtm.ts [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+      description:
+        "Verify DevOps tools according to DevStream config file and state",
+      options: [
+        {
+          name: ["-f", "--config-file"],
+          description: 'Config file (default "config.yaml")',
+          args: {
+            name: "file",
+            template: "filepaths",
+          },
```
Comment (fedeci) [inline src/dtm.ts:265]: Suggestion: if this `--config` only accepts certain file extensions you can use the `@fig/autocomplete-generators` package.

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks, there are a couple nits but overall it looks good!

File: src/dtm.ts (hunk 1 above)
Comment (imxw) [reply src/dtm.ts]: Thanks for your help

**Review by mschrage — APPROVED** [review]
This looks great. Thanks so much for contributing @imxw!

File: src/dtm.ts (hunk 2 above)
Comment (imxw) [reply src/dtm.ts]: Thanks for your help!

**Comment by imxw** [comment]
> Looks like these flags are global:
> 
> ```
>   -f, --config-file string   config file (default "config.yaml")
>   -d, --debug                debug level log
>   -p, --plugin-dir string    plugins directory (default ".devstream")
>   -y, --yes                  apply/delete directly without confirmation
> ```
> 
> Could you add these to the top-level `options` array, and give them all `isPersistent: true`?
> 
> I think this would be the last change! Looks good 🙂

Thanks for your rigor!  With more subcommands, these flags can no longer be used globally. We will move them to the appropriate subcommands in the next version.

**Comment by imxw** [comment]
> @imxw would you be interested in having automatic updates and versioning for this completion spec? I've seen you are using cobra in your CLI and we provide a cobra integration for autogenerating specs. I can open a PR in the next days, WDYT? I am merging this one in the meanwhile, thanks again!

Yes, I've noticed the cobra integration. Once I remove some unnecessary global flags from `dtm`, I'll use it.

**Pull request merged**